src/components/LocationHolder.js

```diff
@@ -3,7 +3,12 @@ import whiteArrow from '../assets/chevron-right-white.svg'
 import { Link } from 'react-router-dom';
 
 export default class LocationHolder extends React.Component {
+ 
     render() {
+        
+        let visibility;
+        this.props.hood ? visibility = "visible" : visibility = "hidden";
```

Comment:
shouldn't this just be 
```suggestion
        const visibility = this.props.hood ? "visible" :  "hidden";
```